src/Components/Aspire_Components_Progress.md

```diff
@@ -10,6 +10,7 @@ As part of the .NET Aspire November preview, we want to include a set of .NET As
 | Microsoft.Data.SqlClient                |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ❌          |        ✅            |         ✅           |              ✅                  |
 | Microsoft.EntityFramework.Cosmos        |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ✅          |        ✅            |         ❌          |              ❌                  |
 | Microsoft.EntityFrameworkCore.SqlServer |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ✅          |        ✅            |         ✅           |              ✅                  |
+| MongoDB.Driver |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ❌          |        ❌            |         ❌           |              ✅                  |
```

Comment:
Are Tracing or Metrics supported at all? Doing a quick search I found https://github.com/jbogard/MongoDB.Driver.Core.Extensions.OpenTelemetry for tracing.

See also https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/127.

Reply:
I've tried to add this package but it is missing in the mirror as well.

Reply:
I'm mirroring it now. It should be done in a few minutes. My hope is that maybe this functionality can get moved into MongoDB Driver itself? Or maybe a library in https://github.com/open-telemetry/opentelemetry-dotnet-contrib.

cc @jbogard - thoughts on moving this functionality to somewhere more "official"?

Reply:
I've added the tracing support please review.

Reply:
Hi @eerhardt! Can we close this one?